### PR TITLE
Add more Visual Studio project files to syntax highlighting

### PR DESCRIPTION
**Pull request by saul**
Sorry - I only realised after the previous PR was merged! Highlighter test PR inbound.

### CODE REVIEW

File: docs/technical/syntax-highlighting.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ We introduced syntax highlighted diffs in [#3101](https://github.com/desktop/des
 
 We currently support syntax highlighting for the following languages.
 
-JavaScript, JSON, TypeScript, Coffeescript, HTML, CSS, SCSS, LESS, VUE, Markdown, Yaml, XML, Objective-C, Scala, C#, Java, C, C++, Ocaml, F#, sh/bash, Swift, SQL, CYPHER, Go, Perl, PHP, Python, Ruby, Clojure
+JavaScript, JSON, TypeScript, Coffeescript, HTML, CSS, SCSS, LESS, VUE, Markdown, Yaml, XML, Objective-C, Scala, C#, Java, C, C++, Ocaml, F#, sh/bash, Swift, SQL, CYPHER, Go, Perl, PHP, Python, Ruby, Clojure, C#/F# projects
```
Comment (say25) [inline docs/technical/syntax-highlighting.md]: Not sure if this is needed or not but this file matches the order of languages in the code so I think it should stay consistent

**Comment by niik** [comment]
> Highlighter test PR inbound.

Don't worry about that. Xml is xml. There's no difference as far as the mode is concerned here. I asked you to add the sample files for F# because there was syntax differences between them and a regular F# file so having samples makes it easy for us to test that they are working.

File: docs/technical/syntax-highlighting.md (hunk 1 above)
Comment (saul) [reply docs/technical/syntax-highlighting.md]: Good catch - I wondered what the order was for.

File: docs/technical/syntax-highlighting.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ We introduced syntax highlighted diffs in [#3101](https://github.com/desktop/des
 
 We currently support syntax highlighting for the following languages.
 
-JavaScript, JSON, TypeScript, Coffeescript, HTML, CSS, SCSS, LESS, VUE, Markdown, Yaml, XML, Objective-C, Scala, C#, Java, C, C++, Ocaml, F#, sh/bash, Swift, SQL, CYPHER, Go, Perl, PHP, Python, Ruby, Clojure
+JavaScript, JSON, TypeScript, Coffeescript, HTML, CSS, SCSS, LESS, VUE, Markdown, Yaml, XML, C#/F# projects, Objective-C, Scala, C#, Java, C, C++, Ocaml, F#, sh/bash, Swift, SQL, CYPHER, Go, Perl, PHP, Python, Ruby, Clojure
```
Comment (niik) [inline docs/technical/syntax-highlighting.md]: I don't think we need this here, it falls under XML. Same as with SVG. The reasoning here is that we don't have any special handling of SVG or `.[cf]sproj` files.

**Comment by shiftkey** [comment]
@saul just needs a merge from `master` to get #3556 integrated

**Comment by saul** [comment]
@niik master is now merged

**Pull request merged**